src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -1113,35 +1168,42 @@ <h2 class="heading-medium">
     </td>
     <td>
       <div>
-        {{ hearingContent.witness.selfWitness }}
+        {{ hearingContent.witness.selfWitness | capitalize }}
       </div>
     </td>
 
   </tr>
-  <tr>
-    <td>
-      <div>
-        Other witnesses
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.witness.noOfOtherWitness }}
-      </div>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <div>Dates unavailable</div>
-    </td>
-    <td>
-      <div>
-        {% for unavailableDate in hearingContent.unavailableDates %}
-        {{ unavailableDate }} <br/>
-        {% endfor %}
-      </div>
-    </td>
-  </tr>
+
+  {% if hearingContent.witness.noOfOtherWitness is not empty %}
+    <tr>
+      <td>
+        <div>
+          Other witnesses
+        </div>
+      </td>
+      <td>
+        <div>
+          {{ hearingContent.witness.noOfOtherWitness }}
+        </div>
+      </td>
+    </tr>
+  {% endif %}
+
+  {% if hearingContent.unavailableDates is not empty  %}
```

Comment:
as above, this should always be there and have 'None' if there are no dates

Reply:
done